code/modules/projectiles/ammunition/ammo_casings.dm

```diff
@@ -255,4 +255,44 @@
 	desc = "A cased high explosive grenade that can only be activated once fired out of a grenade launcher."
 	caliber = "40mm"
 	icon_state = "40mmHE"
-	projectile_type = "/obj/item/projectile/bullet/a40mm"
\ No newline at end of file
+	projectile_type = "/obj/item/projectile/bullet/a40mm"
+
+//FOAM DARTS
+/obj/item/ammo_casing/caseless/foam_dart
+	name = "foam dart"
+	desc = "Its nerf or nothing! Ages 8 and up."
+	projectile_type = "/obj/item/projectile/bullet/reusable/foam_dart"
+	caliber = "foam_force"
+	icon = 'icons/obj/toyguns.dmi'
+	icon_state = "foamdart"
+	var/modified = 0
+
+/obj/item/ammo_casing/caseless/foam_dart/update_icon()
+	....() //to omit adding "-live" to the ammo casing
```

Comment:
[This one](https://github.com/ParadiseSS13/Paradise/blob/a3fdeaa57cdf5b84dc3be2057c0eb0373a41352a/code/modules/projectiles/ammunition.dm#L27) for some reason makes the new ammo casing spawned after the projectile hits the wall have a icon state such as `foamdart-live`.